src/test/java/calculator/domain/NumberListTest.java

```diff
@@ -15,7 +15,7 @@ public class NumberListTest {
         List<Integer> result = List.of(2, 10, 3, 6, 3);
         Numbers numberList = new Numbers(textArray);
 
-        Assertions.assertThat(numberList.findNumber(0)).isEqualTo(2);
+        Assertions.assertThat(numberList.find(0)).isEqualTo(2);
```

Comment:
```suggestion
        assertThat(numberList.find(0, 2)).isTrue();
```

0번째가 2냐라고 검증하라고 시켜볼수도 있을것 같아여.
관점확장의 예시일뿐이며, 꼭 그렇게 해야한다고 코멘트 남기는것은 아닙니다~

최대한 꺼내기보다는 비교의 관점입니다~

Reply:
find는 계산기가 계산할 숫자를 리턴해주는 메소드다 보니 로직상 boolean을 리턴받을 수 없을 것 같다고 생각했습니다! 테스트 코드만을 위한 메소드를 지양하라는 피드백을 받아서 이 메소드는 int를 리턴하고자 합니다. getter 사용을 최대한 안하려고 해보겠습니다